src/actions/index.js

```diff
@@ -0,0 +1,23 @@
+export const createUser = (name, email, password) => {
+  return {
+    type: 'CREATE_USER', 
+    name, 
+    email, 
+    password
+  }
+}
+
+export const signInUser = (email, password) => {
+  return {
+    type: 'SIGN_IN_USER', 
+    email, 
+    password
+  }
```

Comment:
This is great, but we should also add an id in the payload for when we dispatch the `signInUser` action.